lib/internal/http2/core.js

```diff
@@ -1056,8 +1152,10 @@ class Http2Stream extends Duplex {
   }
 
   _read(nread) {
+    if (this._state.destroyed)
+      throw new errors.Error('ERR_HTTP2_INVALID_STREAM');
```

Comment:
why this is happening? It shouldn't. When a stream is destroyed we should push `null`, and this will never be called.